Enabling stacktrace + logrus.FieldLogger + goroutine cause panic.

I'm trying to log errors with stacktraces in goroutines but every time  caught a panic.

Here is a [snippet](https://gist.github.com/cheshir/95583d3e6d30a92c9c2fc4eed5e267e1).

Here is a panic message:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x12c7306]

goroutine 11 [running]:
sentry_err/vendor/github.com/getsentry/raven-go.(*Stacktrace).Culprit(0x0, 0x1533860, 0x0)
	/Users/cheshir/projects/go/src/sentry_err/vendor/github.com/getsentry/raven-go/stacktrace.go:27 +0x26
sentry_err/vendor/github.com/getsentry/raven-go.(*Packet).Init(0xc4204b6480, 0x138a89c, 0x1, 0x0, 0xc42045b618)
```

What I'm doing wrong?

--- a/sentry.go
+++ b/sentry.go
@@ -202,7 +202,9 @@ func (hook *SentryHook) Fire(entry *logrus.Entry) error {
 			packet.Culprit = err.Error()
 		} else {
 			currentStacktrace := raven.NewStacktrace(stConfig.Skip, stConfig.Context, stConfig.InAppPrefixes)
-			packet.Interfaces = append(packet.Interfaces, currentStacktrace)
+			if currentStacktrace != nil {
+				packet.Interfaces = append(packet.Interfaces, currentStacktrace)
+			}
 		}
 	} else {
 		// set the culprit even when the stack trace is disabled, as long as we have an error

--- a/sentry_test.go
+++ b/sentry_test.go
@@ -325,6 +325,16 @@ func TestSentryStacktrace(t *testing.T) {
 		if !strings.HasSuffix(frames[0].Filename, expectedPkgErrorsStackTraceFilename) {
 			t.Error("Stacktrace should be taken from err if it implements the pkgErrorStackTracer interface")
 		}
+
+		// zero stack frames
+		defer func() {
+			if err := recover(); err != nil {
+				t.Error("Zero stack frames should not cause panic")
+			}
+		}()
+		hook.StacktraceConfiguration.Skip = 1000
+		logger.Error(message)
+		<-pch // check panic
 	})
 }
 

